MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -35,14 +35,18 @@ It is also important to note that while this document provides the criteria agai
 # Criteria
 
 ## Frontend
-* If provided, End-to-end tests must be written in an [officially approved technology](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) (3, 4)
+
+Note: Frontend criteria apply to both modules and shared libraries. 
```

Comment:
@vijay-eis and I expressed concern about slowing things down for developers.  But @zburke argued (and @craigmcnally agreed) for equivalence with FE modules:
> My attitude is that code is code. It should all be held to the same standard. TCR process isn’t about “is this good?” but “does this meet established standards?” A shared library looks no different than a new UI app from that angle. Either it has unit tests or it doesn’t. It has a license agreement or it doesn’t. etc.